commerce_coordinator/apps/lms/views.py

```diff
@@ -401,3 +412,121 @@ def post(self, request):
             logger.exception(f"[FirstTimeDiscountEligibleView] HTTP Error: {err}")
 
         return Response({'is_eligible': True})
+
+
+class ProgramPriceView(APIView):
+    """View to get the price of a program."""
+    authentication_classes = (JwtAuthentication,)
+    permission_classes = (AllowAny,)
+    throttle_classes = []
+
+    def _calculate_discounted_price(self, program_offer, program_price):
+        """Calculate the discounted price of the program."""
+        if not program_offer:
+            return program_price
+
+        discounted_price = program_price
+        if program_offer.get("discount_type") == CT_ABSOLUTE_DISCOUNT_TYPE:
+            discounted_price -= program_offer.get("discount_value_in_cents")
+        else:
+            discounted_price -= discounted_price * program_offer.get("discount_value_in_cents") / 10000
```

Comment:
should we use some flooring on the number of decimal points? we faced an issue in past with paypal because of how the floats are handled in languages

Reply:
The use case of this price is to show this to the user so I don't think this will create any issues, but we can use flooring function.